finetune/base_models/bert/table_utils.py

```diff
@@ -79,12 +81,11 @@ def batch_packing(ragged_input, include_mask=True):
 
     # If there is some fast way to sort the rows of a ragged tensor we might consider doing this first.
     col_seq_lens = ragged_input.row_lengths()
-    max_length = tf.reduce_max(col_seq_lens)
+    max_length = tf.minimum(tf.reduce_max(col_seq_lens), 512)
```

Comment:
make these 512 references configurable so that we could easily plugin different models down the road.